lumicks/pylake/detail/confocal.py

```diff
@@ -265,12 +265,23 @@ def timestamps(self) -> np.ndarray:
 
     @property
     def pixelsize_um(self):
-        """Returns a `List` of axes dimensions in um. The length of the list corresponds to the number of scan axes."""
+        """Returns a `List` of axes dimensions in um. The length of the list corresponds to the
+        number of scan axes."""
         return [axes["pixel size (nm)"] / 1000 for axes in self._ordered_axes()]
 
     @property
     def scan_width_um(self):
-        """Returns a `List` of scan widths in um along axes. The length of the list corresponds to the number of scan axes."""
+        """Returns a `List` of scan widths in um along axes. The length of the list corresponds to
+        the number of scan axes."""
+        return [
+            axes["pixel size (nm)"] * axes["num of pixels"] / 1000 for axes in self._ordered_axes()
+        ]
+
+    @property
+    def scan_width_um_ui(self):
+        """Returns a `List` of scan widths as configured in the Bluelake UI. The length of the list
+        corresponds to the number of scan axes. Note that these widths can deviate from the actual
+        scan widths performed in practice"""
         return [axes["scan width (um)"] for axes in self._ordered_axes()]
```

Comment:
To avoid the breaking change, you could deprecate the old one without changing the name and call the new one `.size_um` since its a multidimensional size. The `scan_` part was redundant (`Scan.scan_size_um`) or confusing (`Kymo.scan_size_um`).